src/reference/asciidoc/mongodb.adoc

```diff
@@ -290,3 +290,116 @@ and other attributes that are common across all other inbound adapters (e.g., 'c
 The example above is relatively simple and static since it has a literal value for the `collection-name`.
 Sometimes you may need to change this value at runtime based on some condition.
 To do that, simply use `collection-name-expression` where the provided expression can be any valid SpEL expression.
+
+=== MongoDB Outbound Gateway
+
+The MongoDb Outbound Gateway allows you to query a database by sending a Message to its request channel.
```

Comment:
Would be better to say since which version it is available.

Plus the great addition would be in the  `whats-new.adoc` with the link to this new paragraph.